cpu/esp8266/Makefile.include

```diff
@@ -54,6 +54,10 @@ ifneq (, $(filter littlefs, $(USEMODULE)))
     export RIOT_TEST_TIMEOUT = 200
 endif
 
+ifneq (,$(filter lwip,$(USEMODULE)))
+  CFLAGS += -DTCPIP_THREAD_PRIO=5
```

Comment:
Since you had such a nice comment on why it was `2` for esp32, do you also have one as for where that `5` is coming from?